app/src/test/drawable/label2d_handler.test.ts

```diff
@@ -1161,81 +1158,3 @@ test('Draw label2d list to canvas', () => {
   expect(rect.x2).toEqual(10)
   expect(rect.y2).toEqual(10)
 })
-
-test('Change label ordering', () => {
-  const [label2dHandler] = initializeTestingObjects()
-  Session.dispatch(addPolygon2dLabel(
-    0,
-    -1,
-    [0],
-    [(new PathPoint2D(0, 1, PointType.VERTEX)).toPathPoint(),
-      (new PathPoint2D(1, 1, PointType.VERTEX)).toPathPoint(),
-      (new PathPoint2D(1, 2, PointType.CURVE)).toPathPoint(),
-      (new PathPoint2D(0, 2, PointType.CURVE)).toPathPoint()],
-    true
-  ))
-  Session.dispatch(addPolygon2dLabel(
-    0,
-    -1,
-    [0],
-    [(new PathPoint2D(3, 4, PointType.VERTEX)).toPathPoint(),
-      (new PathPoint2D(4, 4, PointType.VERTEX)).toPathPoint(),
-      (new PathPoint2D(4, 5, PointType.CURVE)).toPathPoint(),
-      (new PathPoint2D(3, 5, PointType.CURVE)).toPathPoint()],
-    false
-  ))
-  Session.dispatch(addPolygon2dLabel(
-    0,
-    -1,
-    [0],
-    [(new PathPoint2D(10, 11, PointType.VERTEX)).toPathPoint(),
-      (new PathPoint2D(11, 11, PointType.VERTEX)).toPathPoint(),
-      (new PathPoint2D(11, 12, PointType.CURVE)).toPathPoint(),
-      (new PathPoint2D(10, 12, PointType.CURVE)).toPathPoint()],
-    true
-  ))
-
-  let state = Session.getState()
-  const labelIds = Object.keys(state.task.items[0].labels)
-  expect(labelIds.length).toEqual(3)
-  expect(state.task.items[0].labels[labelIds[0]].order).toEqual(0)
-  expect(state.task.items[0].labels[labelIds[1]].order).toEqual(1)
-  expect(state.task.items[0].labels[labelIds[2]].order).toEqual(2)
-
-  // Move last label back
-  Session.dispatch(selectLabel(state.user.select.labels, 0, labelIds[2]))
-
-  keyClick(label2dHandler, Key.ARROW_DOWN)
-
-  state = Session.getState()
-  expect(state.task.items[0].labels[labelIds[0]].order).toEqual(0)
-  expect(state.task.items[0].labels[labelIds[1]].order).toEqual(2)
-  expect(state.task.items[0].labels[labelIds[2]].order).toEqual(1)
-
-  // Move first label forward
-  Session.dispatch(selectLabel(state.user.select.labels, 0, labelIds[0]))
-  keyClick(label2dHandler, Key.ARROW_UP)
```

Comment:
We still want to test the change order functions to make sure the labels are drawn in the correct order.

Reply:
I'm confused by this. I thought we were completely removing the order variable, and I also removed the change order functions as a consequence.